### PR TITLE
OCPBUGS-4873: Add audit-version and warn-version labels

### PR DESCRIPTION
**Pull request by tmshort**
**Description of the change:**

Add the PSA labels for audit-version and warn-version

**Motivation for the change:**

OCPBUGS-4873

### CODE REVIEW

**Comment by tmshort** [comment]
The e2e-gcp-serial tests failed due to a slight disruption of services, which has been seen in:

* Other PRs
* Doc change only PRs

/override ci/prow/e2e-gcp-serial

File: manifests/01_namespace.yaml [hunk 1]
```diff
@@ -13,5 +13,7 @@ metadata:
     pod-security.kubernetes.io/enforce: baseline
     pod-security.kubernetes.io/enforce-version: v1.25
     pod-security.kubernetes.io/audit: restricted
+    pod-security.kubernetes.io/audit-version: v1.25
```
Comment (jianzhangbjz) [inline manifests/01_namespace.yaml:16]: One question, as you know, the master branch aligns with OCP 4.13 which is based on K8s 1.26, so any reason to use `1.25` here? Or do we plan to update it to `1.26` after OCP 4.13 rebase to K8s 1.26? Thanks!

Comment (tmshort) [reply manifests/01_namespace.yaml:16]: The plan is to set it to `v1.25` (to match `enforce-version`), back port to 4.12, then update to `v1.26` at the proper time.

**Comment by jianzhangbjz** [comment]
Test passed, details: https://issues.redhat.com/browse/OCPBUGS-4873
/label qe-approved
/lgtm

File: manifests/01_namespace.yaml (hunk 1 above)
Comment (jianzhangbjz) [reply manifests/01_namespace.yaml:16]: Got it, thanks!

**Comment by tmshort** [comment]
/retest

The 400-error test should be available now.

**Comment by tmshort** [comment]
/retest

Looks as though the test is still failing, but the 400 error for the prow test is no longer present

**Pull request merged**